caster/lib/ccr/__init__.py

```diff
@@ -1,16 +1,32 @@
-from bash.bash import Bash
-from core.alphabet import Alphabet
-from core.nav import Navigation
-from core.numbers import Numbers
-from core.punctuation import Punctuation
-from cpp.cpp import CPP
-from csharp.csharp import CSharp
-from haxe.haxe import Haxe
-from html.html import HTML
-from java.java import Java
-from javascript.javascript import Javascript
-from python.python import Python
-from rust.rust import Rust
-from sql.sql import SQL
-from vhdl.vhdl import VHDL
-from prolog.prolog import Prolog
+# new modules should be added in the dictionary
+# the tupple contains the rules which will be imported
+
+command_sets = {
+    "bash.bash": ("Bash", ),
+    "core.alphabet": ("Alphabet", ),
+    "core.nav": ("Navigation", ),
+    "core.numbers": ("Numbers", ),
+    "core.punctuation": ("Punctuation", ),
+    "cpp.cpp": ("CPP", ),
+    "csharp.csharp": ("CSharp", ),
+    "haxe.haxe": ("Haxe", ),
+    "html.html": ("HTML", ),
+    "java.java": ("Java", ),
+    "javascript.javascript": ("Javascript", ),
+    "python.python": ("Python", ),
+    "rust.rust": ("Rust", ),
+    "sql.sql": ("SQL", ),
+    "prolog.prolog": ("Prolog", ),
+    "vhdl.vhdl": ("VHDL", ),
+}
```

Comment:
I would really like to make this list automatically. I just haven't settled on the implementation strategy.